packages/@aws-cdk/aws-glue/lib/connection.ts

```diff
@@ -0,0 +1,193 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core';
+import * as constructs from 'constructs';
+import { CfnConnection } from './glue.generated';
+
+/**
+ * The type of the glue connection
+ *
+ * If you need to use a connection type that doesn't exist as a static member, you
+ * can instantiate a `ConnectionType` object, e.g: `new ConnectionType('NEW_TYPE')`.
+ */
+export class ConnectionType {
+  /** A list of all known `ConnectionType`s. */
+  public static readonly ALL = new Array<ConnectionType>();
+
+  /**
+   * Designates a connection to a database through Java Database Connectivity (JDBC).
+   */
+  public static readonly JDBC = new ConnectionType('JDBC');
```

Comment:
Why not enums? Are these likely to have new values frequently?

Reply:
See my response above